pkgs/development/python-modules/python-smarttub/default.nix

```diff
@@ -26,7 +37,7 @@ buildPythonPackage rec {
   propagatedBuildInputs = [
     aiohttp
     inflection
-    pyjwt
+    pyjwt'
```

Comment:
Don't ever do this. What happens now is that within a closure, you can have two versions of pyjwt. This should absolutely not happen.

The solution is to override the python3 package set that home-assistant uses.

Reply:
Reverted in d9fd85f9fb883f75e2000785116f2480a40a687f and marked the package as conditionally broken. Annoying for users of the package that don't use it through ha, but there is no way around it.